crates/stackable-operator/src/commons/product_image_selection.rs

```diff
@@ -163,6 +160,8 @@ impl ProductImage {
     }
 
     /// The product version is always known without having to resolve the image.
+    /// In the future we might have a more clever version, which let's the operator pick a recommended product version
```

Comment:
I'm highly skeptical about this intention but it's a different discussion.

Reply:
Yep, that's totally fair. We will for sure discuss it before implementing it :sweat_smile: